### PR TITLE
Create database seeder extension

### PR DESCRIPTION
**Pull request by khojianvar**

### CODE REVIEW

File: BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+                new AnalysisType() { Name = "Hematology" },
+                new AnalysisType() { Name = "Microbiology" },
+                new AnalysisType() { Name = "Clinical chemistry" },
+            };
+
+            context.AnalysisTypes.AddRange(analysisTypes);
+            context.SaveChanges();
+        }
+
+        public static void CreateLaboratoryResults(BioMedDbContext context)
```
Comment (Zikirillo2002) [inline BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs:572]: It would be more efficient to convert these public methods to private methods as well.

Comment (khojianvar) [reply BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs:572]: why do you think so?

Comment (Zikirillo2002) [reply BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs:572]: Because these methods are used only in this class and cannot be called in other classes.

Comment (khojianvar) [reply BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs:572]: It`s clear, i correct my mistakes
Thank you

**Review by Zikirillo2002 — APPROVED** [review]

**Pull request merged**